optimade/server/main.py

```diff
@@ -91,30 +106,153 @@ def update_schema(app):
         json.dump(app.openapi(), f, indent=2)
 
 
+def general_exception(
+    request: Request, exc: Exception, **kwargs: Dict[str, Any]
+) -> JSONResponse:
+    tb = "".join(
+        traceback.format_exception(etype=type(exc), value=exc, tb=exc.__traceback__)
+    )
+    print(tb)
+
+    try:
+        status_code = exc.status_code
+    except AttributeError:
+        status_code = kwargs.get("status_code", 500)
+
+    try:
```

Comment:
This can be simplified to a single line using `getattr()`.

Reply:
Note that the try-and-except above can be simplified in the same manner, I just think that the one-liner there would be more confusing than simple when trying to understand the code - so I suggest to leave it alone